src/term.c

```diff
@@ -7180,6 +7194,17 @@ got_code_from_term(char_u *code, int len)
 #endif
 		may_adjust_color_count(val);
 	    }
+	    // when RGB result comes back, it is supported when the result contains an '='
+	    else if (name[0] == 'R' && name[1] == 'G' && name[2] == 'B' && code[9] == '=')
```

Comment:
The `RGB` XTGETTCAP capability returns the number of bits per color channel. This should also check that the returned value is 8.

Reply:
Ah, I wondered what the 8 is for.